src/main/resources/db/changelog/changes/v1.1.0/update-drv-item-details-definition.xml

```diff
@@ -289,6 +289,37 @@
               "valueGetter": "permanent_location_ref_data.jsonb ->> 'name'",
               "visibleByDefault": false
             },
+            {
```

Comment:
Behold, the power of copy/paste! 🙂

Also, I copied this from Holdings. I think I managed to replace all of the holdings-specific stuff with items, but that's definitely something to keep an eye out for in this PR

Reply:
Extreme efforts